zipline-gradle-plugin/src/test/kotlin/app/cash/zipline/gradle/ZiplinePluginTest.kt

```diff
@@ -325,7 +325,7 @@ class ZiplinePluginTest {
     val result = createRunner(projectDir, "clean", taskName).buildAndFail()
     assertThat(result.output).contains(
       """
-      |Zipline API file is incomplete. Run :ziplineApiDump to update it.
+      |Zipline API file is incomplete. Run zipline-api-dump to update it.
```

Comment:
Not sure how I feel about this. The CLI’s instructions recommend the CLI command to re-run, but this could be confusing for Gradle users since it wants `:ziplineApiDump` and not `zipline-api-dump`.

Reply:
I pass in the command to report in failure messages so that I can vary it between raw CLI usage and Gradle usage.

Reply:
Done. Good idea.